ipatests/test_integration/test_server_del.py

```diff
@@ -309,3 +309,12 @@ def test_forced_removal_of_master(self):
             self.replicas[0], self.master.hostname,
             ignore_last_of_role=True
         )
+
+    def test_removal_with_verbose_option(self):
+        """
+        Test remove master with verbose option
+        """
+        tasks.install_master(self.master)
+        result = tasks.uninstall_master(self.master, verbose=True)
```

Comment:
Travis is failing because of uninstall_master does not have return we should change it :)
Assigning to function call which doesn't return